### ISSUE TITLE
Don't observe any result from Run Recent Command

### ISSUE THREAD

**Issue by bamurtaugh** [microsoft/vscode#194171]
Testing #193852

Testing on Windows 11.

- `alt+f1` opens terminal accessibility help dialog correctly:

![Image](https://github.com/microsoft/vscode/assets/25310137/81624354-dc49-4cd3-bdaf-b4c95f929e4e)

- Explore options, decide I want to try Run Recent Command (`Control+Alt+R`)
- `alt+f2` to open terminal accessible view, opens correctly:

![Image](https://github.com/microsoft/vscode/assets/25310137/4a1ced9f-e542-4efd-9b7b-ac22e56f62c4)

- Try `Control+Alt+R`
- ❓ I don't observe anything happen - but maybe I'm misunderstanding what should happen or how I should set this command up?

**Comment by meganrogge** [comment]
Sorry this was not clear. Perhaps I should specify which commands only apply when focus is in the terminal? 

Perhaps that should also work when the focus is in the accessible view... I will think about that.

**Comment by meganrogge** [comment]
To verify, focus a terminal in screen reader mode. open the terminal accessibility help dialog with `alt+f1` and check which keybinding to use for `run recent command`. 

open the terminal accessible view by focusing the terminal and using `alt+f2.` use that key binding. make sure it runs whichever command you select.

**Comment by bamurtaugh** [comment]
I think I'm still missing something in my understanding. Steps I took:

- Open VS Code repo in latest VS Code Insiders
- Open and focus terminal
- `alt+f1`, the terminal accessibility help dialog opens
- I note that `ctrl+alt+r` will run recent command, and `alt+uparrow` will get me to previous commands
- Open terminal accessible view by focusing terminal and using `alt+f2`
- `alt+uparrow` to get to a previous command (this worked well)
- Once I've gotten to a previous command I'd like to run, I try `ctrl+alt+r` but again don't observe anything happened:

![image](https://github.com/microsoft/vscode/assets/25310137/504add79-7615-46da-aedb-6ff5e9718628)

**Comment by meganrogge** [comment]
Apologies, Brigit, it appears there's an old bug resurfacing in which the keybinding is not listed correctly in this dialog. 

On windows, run recent command is invoked with `ctrl+r` when in screen reader mode. 

On mac, with screen reader mode, it's `cmd+r`.

**Comment by meganrogge** [comment]
I'm going to make an issue about that.

**Comment by bamurtaugh** [comment]
Ah yay, `ctrl+r` worked well for me, thank you! I'll hold off on labeling as `verified` if you'd also like to use this issue to track fixing the keybinding bug.

**Comment by meganrogge** [comment]
Actually, I think I'll reopen that other issue. Since the issue was the command not working, it would be great if you'd verify this and I'll link the other 😄 . Thanks

**Comment by meganrogge** [comment]
here's the issue where we're tracking that https://github.com/microsoft/vscode/issues/175130
